sdk/test/action-handler-test.js

```diff
@@ -130,7 +145,6 @@ describe('ActionHandler', () => {
   });
 
   // synchronous handlers
```

Comment:
Hmm, removed spacing as well, try adding it back.

```suggestion
  // synchronous handlers

```